### ISSUE TITLE
"verdi code show" no-argument error message ambiguous

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida_core#698]
Using the latest development code, I have the following (problematic) behaviour

```
(aiidapy)aiida_test@theospc22:~$ verdi -p htsqla9.5_v2 code show
09/15/2017 06:09:11 PM, alembic.runtime.migration: [INFO] Context impl PostgresqlImpl.
09/15/2017 06:09:11 PM, alembic.runtime.migration: [INFO] Will assume transactional DDL.
after 'code show' there should be one argument only, being the code id.
```

**Comment by DropD** [comment]
`verdi code show` expects a positional argument, the error message should probably say "exactly one argument" instead of "one argument only".

I will rename this issue accordingly

**Comment by ltalirz** [comment]
Fixed in #1655
